RecoCTPPS/Configuration/python/recoCTPPS_sequences_cff.py

```diff
@@ -6,8 +6,9 @@
 from RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteProducer_cff import ctppsLocalTrackLiteProducer
 from RecoCTPPS.PixelLocal.ctppsPixelLocalReconstruction_cff import *
 
-from Geometry.VeryForwardGeometryBuilder.ctppsIncludeAlignmentsFromXML_cfi import *
-ctppsIncludeAlignmentsFromXML.RealFiles = cms.vstring("Alignment/CTPPS/data/RPixGeometryCorrections.xml")
+from CondFormats.CTPPSReadoutObjects.ctppsRPAlignmentCorrectionsDataESSourceXML_cfi import *
+ctppsRPAlignmentCorrectionsDataESSourceXML.RealFiles = cms.vstring("Alignment/CTPPS/data/RPixGeometryCorrections.xml")
+ctppsRPAlignmentCorrectionsDataESSourceXML.verbosity = 10
```

Comment:
the verbosity setting looks like an omission from debugging stage of the developments.
Please check and possibly reduce.
Please clarify why the RealFiles can not be set in the ```ctppsIncludeAlignmentsFromXML_cfi``` file.
This looks like a dangerous approach. If this file is loaded again by someone later, the values will be overridden.

Reply:
Yes, the verbosity statement is a left over, I will fix it.

In principle, the alignment file can be specified in the cfi file. However, up to now we have preferred to set all the conditions in a single place, e.g. here:
https://github.com/CTPPS/cmssw/blob/proton_reco_10_4_X/RecoCTPPS/Configuration/python/recoCTPPS_sequences_cff.py#L12
Obviously this will change once we switch to loading conditions data from DB.

Reply:
Verbosity statement removed.